### ISSUE TITLE
BaseCsrfVerifier expects the field name to be "csrf-token"

### ISSUE THREAD

**Issue by NateNjuguna** [skipperbent/simple-php-router#432]
### Description

The library fails to verify a csrf token in any submitted form on any route. The field is submitted as **"csrf_token"** as said in the docs [here](https://github.com/skipperbent/simple-php-router/#getting-csrf-token):

> The default name/key for the input-field is csrf_token and is defined in the POST_KEY constant in the BaseCsrfVerifier class.

 but a TokenMistmatchException is thrown. However changing to ***"csrf-token"*** solves the issue as the BaseCsrfVerifier middlware expects this instead as shown in the snippet below:
```php
...
class BaseCsrfVerifier implements IMiddleware
{
    const POST_KEY = 'csrf-token';
...
```

### Steps to reproduce the error

1. Add the following csrf verifier:

```php
...
use Pecee\Http\Middleware\BaseCsrfVerifier;
...
SimpleRouter::csrfVerifier(new BaseCsrfVerifier());
...
```

2. Submit a form with a csrf token as *csrf_verifier* to `/login` in browser.

3. `TokenMismatchException` is thrown by library.

### Route and/or callback for failing route

*Route:*

```php
SimpleRouter::post('/login', 'Auth@postLogin');
```

*Callback:*

```php
public function postLogin() {
    echo 'Fine';
}
```

### Debug info

```php

N/A

```

**Comment by skipperbent** [comment]
Hi @NateNjuguna .

Thanks for the detailed description of the issue, I really appreciate it.

It seems like the `POST_KEY` constant was changed for some reason from `csrf_token` to `csrf-token` like you're describing, however the change was never reflected in the documentation.

I've changed the value back to `csrf_token`. The fix will be available in the next upcoming release.

Sorry for the inconvenience. 

/ Simon
